src/pages/ProductPage.jsx

```diff
@@ -52,83 +53,101 @@ const ProductPage = () => {
+              address={"Berlin, Germany"}
+              guests={{ key: "guest", value: 3 }}
+              bedrooms={{ key: "bedroom", value: 1 }}
+              beds={{ key: "bed", value: 3 }}
+              baths={{ key: "bath", value: 1 }}
+              starGrade={4.65}
+              reviews={23}
+            />
+
+            <FavoriteStay />
+            <HostSummary
+              hostName="Raus"
+              hostingDuration={1}
+              role="Superhost"
+              profilePicUrl={hostImage}
+            />
+            <ProductHighlight highlights={highlights} />
+            <hr />
+            <ProductDescription
+              descriptionPlace="The apartment consists of a large living room, a private, large bathroom with a bathtub and a high space, which is suitable for the storage of luggage and is accessed by a staircase.accessed by a staircase"
+              descriptionSpace="The apartment starts from a quiet courtyard and has its own entrance, which does not depart from the general stairwell."
+              guestAccess="You have access to all areas of the flat. The flat has its own entrance."
+              otherThings="Do not smoke in rooms!"
+            />
+          </div>
+          <div className={styles.ReservationCard}>
+            <ReservationCard />
+          </div>
+        </div>
+        <hr className={styles.separator} />
+        <Reviews
+          name={"Julia"}
+          picture={
+            "https://a0.muscache.com/im/pictures/user/5c7af12d-86a7-48f9-a58b-2dfcb88399b7.jpg?im_w=240"
+          }
+          rating={5}
+          reviewText={
+            "It was really super relaxing days with lots of peace and quiet. So if you need a little break, I can definitely recommend the tiny house."
+          }
+          date="2024-09-15"
+        />
+        <MapView
+          mapViewSampleImg={mapViewSampleImg}
+          address="Königslutter am Elm, Niedersachsen, Germany"
+          addressDescription="In the midst of a diverse nature park, you will find yourself surrounded by hilly landscapes covered with dense forests, moors, gorgeous heaths and salt marshes. The surroundings invite you to explore them at any time of the year: hike through one of the largest beech forests in the region, where you will occasionally encounter rare forest dwellers, go mushroom hunting in a popular hiking area nearby, or take a bike ride to a vantage point overlooking aln the midst of a diverse nature park, you will In the midst of a diverse nature park, you will find yourself surrounded by hilly landscapes covered with dense forests, moors, gorgeous heaths and salt marshes. The surroundings invite you to explore them at any time of the year: hike through one of the largest beech forests in the region, where you will occasionally encounter rare forest dwellers, go mushroom hunting in a popular hiking area nearby, or take a bike ride to a vantage point overlooking aln the midst of a diverse nature park, you will"
```

Comment:
It would be great to extract this part in a separate variable as well.

Reply:
It is related to another ticket.